example/babel.config.js

```diff
@@ -14,6 +14,6 @@ module.exports = {
         },
       },
     ],
-    [path.resolve('../plugin')],
+    [path.resolve('../dist/babel')],
```

Comment:
potentially we may be able to use package exports to still reference the babel plugin in actual projects by `@crherman7/rechunk/plugin` or `@crherman7/rehcunk/babel` tbd tho.